src/module-1/card/style.css

```diff
@@ -0,0 +1,105 @@
+button {
+   background: #6f64f8;
+   color: #ffffff;
+   border: 0;
+}
+
```

Comment:
no empty lines
or insert it before every selector to keep the same code style for whole file